src/serlo-editor/plugins/serlo-table/editor.tsx

```diff
@@ -90,15 +90,7 @@ export function SerloTableEditor(props: SerloTableProps) {
       <>
         {props.focused || nestedFocus ? <SerloTableToolbar {...props} /> : null}
         <div className="flex">
-          <div
-            className="flex flex-col"
-            onClick={(e) => {
-              // another hack to make focus ux at least ok
-              const target = e.target as HTMLDivElement
-              const hackDiv = target.querySelector('.hackdiv') as HTMLDivElement
-              hackDiv?.focus()
-            }}
```

Comment:
if this is removed a click inside a cell (but not on the slate field itself) nothing happens any more.
But solving this problem by removing the margin around and adding a padding to the slate field is probably the better solution.

Reply:
postponed work on table plugin